source/common/json/config_schemas.cc

```diff
@@ -464,6 +464,62 @@ const std::string Json::Schema::TCP_PROXY_NETWORK_FILTER_SCHEMA(R"EOF(
   {
       "$schema": "http://json-schema.org/schema#",
       "type" : "object",
+      "definitions": {
```

Comment:
Would prefer to not duplicate the schema. Can we carve this out into a new schema object and validate at a lower level?